src/storage/admin/StatsTask.cpp

```diff
@@ -219,7 +216,10 @@ nebula::cpp2::ErrorCode StatsTask::genSubTask(GraphSpaceID spaceId,
     }
     edgeIter->next();
   }
-
+  while (vertexIter && vertexIter->valid()) {
+    spaceVertices++;
+    vertexIter->next();
+  }
```

Comment:
This is not handled correctly

Reply:
Please ignore the comments above, I see that `vertexkey` is written by default in AddVerticesProcessor.